@shared/api/internal.ts

```diff
@@ -14,7 +14,7 @@ import { APPLICATION_STATE } from "../constants/applicationState";
 import { sendMessageToBackground } from "./helpers/extensionMessaging";
 import { getIconUrlFromIssuer } from "./helpers/getIconUrlFromIssuer";
 
-const TRANSACTIONS_LIMIT = 15;
+const TRANSACTIONS_LIMIT = 100;
```

Comment:
we had this limit on because we had limited space in the old UI. Having a long list of transactions shouldn't be an issue now